### PR TITLE
Add ability to auto downgrade tag release

### PR DESCRIPTION
**Pull request by driftingly**
This PR adds the ability to downgrade the PHP version and tag a new release.

See: 

- https://tomasvotruba.com/blog/how-to-release-php-81-and-72-package-in-the-same-repository/
- https://github.com/TomasVotruba/cognitive-complexity/blob/main/.github/workflows/downgraded_release.yaml
- https://github.com/symplify/config-transformer/blob/main/.github/workflows/downgraded_release.yaml

### CODE REVIEW

File: build/composer-php-72.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "driftingly/rector-laravel",
+    "type": "rector-extension",
+    "license": "MIT",
+    "description": "Rector upgrades rules for Laravel Framework",
+    "require": {
+        "php": "^7.2 || 8.0.*"
```
Comment (TomasVotruba) [inline build/composer-php-72.json:7]:
```suggestion
        "php": "^7.2 || ^8.0"
```

This allows more project to use it :)

File: build/composer-php-72.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "psr-4": {
+            "RectorLaravel\\": "src"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "RectorLaravel\\Tests\\": "tests"
+        },
+        "classmap": ["stubs"]
+    },
```
Comment (TomasVotruba) [inline build/composer-php-72.json:19]: In the relesae package, there should be no tests

```suggestion
    "autoload-dev": {
        "classmap": ["stubs"]
    },
```

File: build/composer-php-72.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "allow-plugins": {
+            "cweagans/composer-patches": true,
+            "rector/extension-installer": true,
+            "phpstan/extension-installer": true
+        }
+    }
```
Comment (TomasVotruba) [inline build/composer-php-72.json:28]: This part can be droped as well IMO

**Comment by TomasVotruba** [comment]
Looks good :) how does it perform so far?

**Comment by TomasVotruba** [comment]
Looks good :) how does it perform so far?

**Comment by driftingly** [comment]
I looks like it's working now.  Thanks. @TomasVotruba 🙌

**Comment by TomasVotruba** [comment]
Amazing work 👏

**Pull request merged**